com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/utils/FileSystemUtils.java

```diff
@@ -46,6 +46,13 @@
 @UtilityClass
 public class FileSystemUtils {
   private final List<String> ALLOWED_EXTENSIONS = Arrays.asList("cpy", "cbl", "cobol", "cob");
+  private final String COPYBOOKS_FOLDER = ".copybooks";
+
+  /** @return the representation os based of the FS separator */
+  public static String filesystemSeparator() {
+    return FileSystems.getDefault().getSeparator();
```

Comment:
Extracting from the javadoc of this method :
`In the case of the default provider, this method returns the same separator as {@link java.io.File#separator}.`
So, you may use just `File.separator` here

Reply:
When I introduced the ```filesystemSeparator()```I decided to use the NIO version instead of the Java.IO ```FileSystems.getDefault().getSeparator();```